resources/js/Pages/Collections/Components/CollectionsFilter/CollectionsFilterPopover.tsx

```diff
@@ -45,8 +45,12 @@ export const CollectionsFilterPopover = ({
         }
     }, [debouncedQuery]);
 
-    const allNetworksSelected =
-        availableNetworks.filter((network) => network.collectionsCount > 0).length === selectedChainIds.length;
+    const isNetowrkFilterActive =
+        isLoading ||
```

Comment:
the downside of this check is that you click a network and it will show the dot for a brief moment before ending up in the loading state where it gets hidden only to show again afterwards, resulting in a flickering when changing filters